community/knative/README.md

```diff
@@ -0,0 +1,200 @@
+# Knative
+
+This chart installs Knative components for Build, Serving, Eventing and Eventing-Sources.
+
+## Introduction
+
+- Visit [Knative Build](https://github.com/knative/build/blob/master/README.md) for more information about Build.
+- Visit [Knative Eventing](https://github.com/knative/eventing/blob/master/README.md) for more information about Eventing.
+- Visit [Knative Eventing Sources](https://github.com/knative/eventing-sources/blob/master/README.md) for more information about Eventing Sources.
+- Visit [Knative Serving](https://github.com/knative/serving/blob/master/README.md) for more information about Serving.
+
+## Chart Details
+
+This chart is comprised of multiple subcharts which is illustrated in the structure below:
+```
+knative                         (default)
+├── build                       (default)
+├── eventing
+|   ├── gcpPubSubProvisioner
+│   ├── inMemoryProvisioner
+|   ├── kafkaProvisioner
+│   └── natssProvisioner
+├── serving                     (default)
+│   └── monitoring              (default)
+│       ├── elasticsearch       (default)
+│       ├── prometheus          (default)
+|       ├── zipkin
+│       └── zipkinInMem         (default)
+└── eventingSources
+    ├── camel
+    ├── eventDisplay
+    └── gcpPubSub
+```
+Disabling a chart will disable all charts below it in the chart structure. When enabling a subset of charts note that the parent charts are prerequisites and must be installed previously or in conjunction.
+
+## Prerequisites
+- Requires kubectl v1.10+.
+- Knative requires a Kubernetes cluster v1.11 or newer with the
+[MutatingAdmissionWebhook admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-on-an-admission-controller)
+enabled.
+- Istio - You should have Istio installed on your Kubernetes cluster. If you do not have it installed already you can do so by running the following commands:
+```bash
+$ kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.1/istio-crds.yaml
+$ kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.1/istio.yaml
+```
+or by following these steps:
+[Installing Istio](https://www.knative.dev/docs/install/knative-with-any-k8s/#installing-istio)
+
+
+## Installing the Chart
+
+Please ensure that you have reviewed the [prerequisites](#prerequisites).
+
+1. Install Knative crds
+```bash
+$ kubectl delete -f knative/all-crds.yaml
+```
+
+2. Install the chart using helm cli:
+
+```bash
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
```

Comment:
I think this can be removed, if your signed into the cluster using the cli only the helm install is needed?